tech/languages/nodejs/modules.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Modules
+page: nodejs
+order: 2
+---
+
+# Modules
+
+## Installation
+
+To install additional modules from Fedora repositories, use:
+
+```
+$ sudo dnf install nodejs-<module-name>
+```
+
+or 
+
+```
+$ sudo dnf install 'npm(module-name)'
+```
+
+For example, to install express, you need to type:
+
+```
+$ sudo dnf install nodejs-express
+```
+
+or 
+
+```
+$ sudo dnf install 'npm(express)'
+```
+
+Most modules are prefixed with 'nodejs-'. There are, however, a few exceptions (for example mocha), which are not prefixed.
+
+## Using modules
+
+npm allows using require() only on locally installed modules. If you want to require() modules installed by dnf, you need to type:
+
+```
+$ npm link express
+```
+
+or to load global modules by default, you can set $NODE_PATH as follows: 
+
+```
+export NODE_PATH=/usr/lib/node_modules
+```
+
+## Missing modules
```

Comment:
Although it's nice to recommend joining the sig, I would mention the upstream module installation as well (perhaps above in the installation section = just to make clear that there are both options).